dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl/server.xml

```diff
@@ -76,14 +76,11 @@
         <attributesCache size="4000" timeout="1200ms" enabled="true" sizeLimit="2000"/>
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
-      <failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.6.name}" port="${ldap.server.6.ssl.port}"/>
-        </failoverServers>
 	</ldapRegistry>
 	
     <federatedRepository maxSearchResults="2000" searchTimeout="400000">
-        <primaryRealm name="ADSSLRealm">
```

Comment:
Change this back, as well as change it back in the test. Lets not change anything we don't need to. The goal here is to NOT change the test as much as possible.